server/pulp/plugins/util/misc.py

```diff
@@ -142,3 +143,89 @@ def clear_directory(path, skip_list=()):
 
         elif os.path.isfile(entry_path):
             os.unlink(entry_path)
+
+
+def common_path(dirs):
```

Comment:
need doc block. It would be helpful to explain what exactly this is doing beyond what ``os.path.commonprefix`` does and why.